poddefaults.go

```diff
@@ -47,17 +46,6 @@ func (s *server) GetPodDefaults(w http.ResponseWriter, r *http.Request) {
 		if desc == "" {
 			desc = pd.Name
 		}
-
-		// Ignore the protected-b poddefault
-		labelMapped := reflect.ValueOf(pd.Spec.Selector.MatchLabels).MapKeys()[0].String()
-
-		if labelMapped != "notebook.statcan.gc.ca/protected-b" {
-			resp.PodDefaults = append(resp.PodDefaults, poddefaultresponse{
-				PodDefault:  *pd,
-				Label:       labelMapped,
-				Description: desc,
-			})
```

Comment:
As noted in slack, think this should be kept? Similar to how it was before this https://github.com/StatCan/jupyter-apis/commit/6c4b579c25ac1904fb6087e0f1cd326a8e04376e#diff-30c3cd28cd4fb301331e7d559ed038961b7d46a282bb3c9e26570e82e49e8e64L51

Everything else looks fine to me, just need to test live